src/Atmos/Model/remainder.jl

```diff
@@ -1,3 +1,5 @@
+export RemainderModel
```

Comment:
this entire struct / file should probably be moved into DGmethods as nothing here is atmosphere specific. if you remove L35 (which doesn't seem to be used for anything), this becomes entirely generic